### ISSUE TITLE
Checkliste item editing flaky

### ISSUE THREAD

**Issue by mkae** [dmfs/opentasks#299]
It more than seldomly happened to me that I lost text I was just entering into a new checklist item. I can't say what exactly triggers it, but it's so easy to loose all your input of a new and still unsaved checklist item... It could be connected to some touchscreen signals which get misinterpreted as deleting big chunks of text...

At least I can say that the likelihood to loose text in these edit fields is lower if I am typing only slowly and very carefully, that's for sure!

**Comment by korelstar** [comment]
I think that I suffered under this issue, too: I was typing some lines into a checklist item in the `ViewTaskActivity`. Then, the keyboard suddenly closed and the text of the item was reset to an old state, i.e. my changes were lost.

I have no idea, what causes this. Is there a kind of auto-save which may work wrong?

**Comment by mkae** [comment]
Yeah, that's exactly it. It has to do with the interpretation of touchscreen events. Certain accidental erratic touches, swipes or moves lead to the app forgetting all changes made up to that moment in time since the last save happened.

**Comment by mkae** [comment]
I believe that there seems to be some timeout for keyboard inputs. I noticed that I didn't touch m smartphone's keyboard at all and suddenly the changes made earlier were reverted.

**Comment by MaDill** [comment]
I can see this issue also from time to time. I use the app on an BQ Aquaris X with Android 8.1

**Comment by bjornfor** [comment]
I'm affected by this too. Very annoying. I've (sadly) grown the habit of typing only a few words before pressing the "back" key to save my work before going back in and type some more words. This happens on Samsung S7, LG V30, Sony Z5 Compact. Stock Android. Latest OpenTasks version. All synced with latest DAVdroid against my Nextcloud v13 instance (same with Nextcloud v12).

I've not seen any other app having this problem. It's as if opentasks out of nowhere suddenly decides to close the virtual keyboard and does not allow it to be reopened for a certain time period.

**Comment by dmp1ce** [comment]
Same issues here on Pixel (Android version 9) with DAVdroid synced to Nextcloud.
